src/plugin.js

```diff
@@ -61,6 +61,13 @@ const initPlugin = function(player, options) {
   const resetMonitor = function() {
     window.clearTimeout(monitor);
     monitor = window.setTimeout(function() {
+      // if using Flash then make sure its API is available
+      if (/swf/i.test(player.tech_.el().data) &&
```

Comment:
Can we get at this without using `tech_`? That's supposed to be a private property.